opentreemap/treemap/templates/treemap/partials/detail_external_link.html

```diff
@@ -0,0 +1,16 @@
+{% load instance_config %}
+{% load util %}
+
+{% if last_effective_instance_user|has_permission:'view_external_link' %}
+{% if last_instance|instance_config:'externalLink.text' %}
+{% if last_instance|instance_config:'externalLink.url' %}
+{% if feature|validate_external_link %}
+    <a href="{{ feature|get_external_link }}"
+       title="{{ last_instance|instance_config:'externalLink.text' }}">
+        {{ last_instance|instance_config:'externalLink.text' }}
+    </a>
+    <hr>
+{% endif %}
+{% endif %}
+{% endif %}
+{% endif %}
```

Comment:
Is there a reason why the logic for whether to display the external link is in the template and several custom filters rather than the view? 

If not, we (and most Django developers) consider it preferable to use view logic rather than template logic. Such as:
```
{% if external_link.should_show %}
    <a href="{{ external_link.url }}" title="{{ external_link.text }}">{{ external_link.text }}</a>
    <hr>
{% endif %}
```
Some reasons:
* Python code is more flexible and readable than template code
* Logic is more localized in the view than in template filters